core/app/views/spree/order_mailer/inventory_cancellation_email.html.erb

```diff
@@ -19,7 +19,7 @@
           </tr>
         <% end %>
       </table>
-      <p>
+      <p></p>
```

Comment:
[After examining the commit](https://github.com/solidusio/solidus/commit/67914228162aaf3cedb4bec2aeaf00fe608a2957) and the related PR that introduced the `<p>` tag, I don't think we need it.

Reply:
Yea, I think we can simply remove this.